Patches/Vanilla Factions Expanded - Tribals/ThingDefs_WeaponsRanged.xml

```diff
@@ -0,0 +1,165 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Vanilla Factions Expanded - Tribals</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+
+				<!-- === Throwingspike === -->
+	
+				<li Class="PatchOperationAdd">
+					<xpath>Defs</xpath>
+					<value>
+
+						<!-- This dummy parent just provides a workaround so VFE - Tribal can be loaded before CE. -->
+						<ThingDef ParentName="BaseWeaponAndAmmoNeolithic" Name="SpikeBase_VFET" Abstract="True" />
+
+						<ThingDef ParentName="BasePilumProjectile">
+							<defName>Tribal_Spike_Thrown</defName>
+							<label>throwspike</label>
+							<projectile Class="CombatExtended.ProjectilePropertiesCE">
+								<damageAmountBase>14</damageAmountBase>
+								<speed>10</speed>
+								<armorPenetrationBlunt>6.44</armorPenetrationBlunt>
+								<armorPenetrationSharp>3.5</armorPenetrationSharp>
```

Comment:
That seems rather... high for hand thrown wooden spikes.

Reply:
Good catch--I copy-pasted the patch from the VFE: Classical pilum, and apparently my commit with some balance tweaks didn't make it in.